docs/content/envoy-authorization.md

```diff
@@ -276,7 +316,9 @@ spec:
 kubectl apply -f deployment.yaml
 ```
 
-> The `proxy-init` container installs iptables rules to redirect all container traffic through the Envoy proxy sidecar. More information can be found [here](https://github.com/open-policy-agent/contrib/tree/master/envoy_iptables).
+> The `proxy-init` container installs iptables rules to redirect all container
+  traffic through the Envoy proxy sidecar. More information can be found
+  [here](https://github.com/open-policy-agent/contrib/tree/master/envoy_iptables).
```

Comment:
It would be helpful to add a note about the `-w` flag here.

Reply:
Done!

Reply:
Looks good. Thanks !